src/effects/VST3/module_linux.cpp

```diff
@@ -0,0 +1,348 @@
+//-----------------------------------------------------------------------------
```

Comment:
I'm skipping review of the three module* files of course, assuming they are cut-and-pasted from third party code.